src/backend/genes.js

```diff
@@ -3,15 +3,15 @@ import { fetchJson } from '.';
 
 const prefix = 'gene/';
 
-export const fetchGene = async ({ id }) => {
+export const fetchGeneDetails = async ({ id }) => {
   const url = server + prefix + id;
-  return fetchJson(url, true);
+  return fetchJson(url);
 };
 
-export const fetchGenes = async ({ search }) => {
+export const fetchGeneSearch = async ({ string }) => {
```

Comment:
Do you mind using a more descriptive parameter name instead of `string`? It is too close to `String` in JS.

Reply:
What would you suggest.

Note that the way I have the state set up is very hierarchical, moreso than hetmech was. So it's like: `state.gene.searches.string`. So I would put something like `searchString`, but `search` is already there. Maybe `input`?

Reply:
I think `queryString` or even `str` would be better than `string`.

Reply:
How about just `query`. Then we would have:

`state.gene.searches.query`
and
`state.gene.searches.results`

Reply:
Sure. That seems good!